vllm/model_executor/models/__init__.py

```diff
@@ -37,7 +37,8 @@
     "QWenLMHeadModel": ("qwen", "QWenLMHeadModel"),
     "RWForCausalLM": ("falcon", "FalconForCausalLM"),
     "StableLMEpochForCausalLM": ("stablelm", "StablelmForCausalLM"),
-    "YiForCausalLM": ("yi", "YiForCausalLM")
+    "YiForCausalLM": ("yi", "YiForCausalLM"),
+    "OrionForCausalLM": ("orion", "OrionForCausalLM")
```

Comment:
Could you sort this by alphabetical order?

Reply:
thanks ! I'll change it